addons/web_editor/static/src/scss/web_editor.variables.scss

```diff
@@ -630,6 +630,35 @@ $o-color-extras-nesting-selector: '&, .o_colored_level &';
     @return $color1;
 }
 
+// Given a primary color (and an eventually secondary one),the function returns
+// a sass map defining a basic odoo palette. Eventual overrides (eg. 'menu': 4)
+// are passed by the arbitrary argument.
+@function o-palette($-primary, $-secondary: null, $-overrides...) {
```

Comment:
> Not sure we support spread operator in our sass version, did you check?

I cant find the changelog that introduces rest parameters in sass (or libsass) :/ ...
Seems to me that it has been introduced a very long time ago, I found an article dated back to 2016.
https://una.im/sass-es2015/
Have you experienced issues with that? Btw I'm using `libsass  0.17.0`, same a `requirements.txt`

Reply:
> Have you experienced issues with that?

No, but as no-one ever used that, I just don't know if we support it.

> Btw I'm using libsass 0.17.0, same as requirements.txt

I guess it's ok to use then but I would still not use it in this case. It makes no sense slowing down the function for the hypothetical case of receiving multiple maps while the caller can (and should) just send an unique map with all its overrides. There are only downsides to this:
- Slower (I mean, even in the case we receive no override)
- More code
- Less readable function
- Less readable call if someone decides to send 5 maps of 1 param instead of 1 map of 5 params

Reply:
@qsm-odoo , spread operator replaced with normal argument 👍